.github/workflows/diff-report.yml

```diff
@@ -256,7 +262,13 @@ jobs:
 
       - name: Set message
         id: out
-        run: echo "::set-output name=message::$(cat message)"
+        run: |
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          {
+            echo "message<<$EOF"
+            cat message
+            echo "$EOF"
+          } >> "$GITHUB_OUTPUT"
```

Comment:
I think we need to make it obvious to others that may not be experienced with bash what is going on here. As GitHub had their special `set-output` command that we replaced, we should create a script to use for appending multiple lines to workflow output. Please create an issue on this, I propose we do something like `./append-to-github-output.sh message`.

This will also help in future maintainence/debugging efforts, since we can check for existence of `message`, etc.

Reply:
Agreed. Opened at https://github.com/checkstyle/checkstyle/issues/12954.